SpatialGDK/Source/SpatialGDK/Public/SpatialView/AuthorityRecord.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "SpatialView/EntityComponentId.h"
+#include "Containers/Array.h"
+
+namespace SpatialGDK
+{
+
+ // A record of authority changes to entity-components.
+ // Authority for an entity-component can be in at most one of the following states:
+ //  Recorded as gained.
+ //  Recorded as lost.
+ //  Recorded as lost-temporarily.
+ //
+ // The possible state transitions for a given entity-component are
```

Comment:
I'd remove the documentation here of the possible states, since you have the explicit trigger->transition below.